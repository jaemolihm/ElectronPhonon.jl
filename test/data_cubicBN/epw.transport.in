--
&inputepw
  prefix      = 'bn'
  amass(1)    = 28.086
  outdir      = 'temp/'
  dvscf_dir   = './save/'
  etf_mem     = 0

  elph        = .true.
  kmaps       = .true.  ! if true, read kmaps from file
  epbread     = .false.
  epwread     = .true.

  lpolar      = .true.
  lifc        = .true.
  use_ws      = .true.
  vme         = .true.

  export_julia = .false.

  scattering  = .true.
  iterative_bte = .true.
  mob_maxiter = 20
  broyden_beta= 1.0
  int_mob     = .false.
  carrier     = .true.
  ncarrier    = -1E15
  mp_mesh_k   = .false.

  efermi_read = .true.
  fermi_energy= 11.4 ! eV
  fsthick     =  0.5 ! eV

  ! Valence band maximum    =  11.477147 eV
  ! Conduction band minimum =  19.670311 eV

  degaussw    = 0.08 ! eV
  nstemp      = 2
  temps       = 200.0 300.0 400.0 ! K

  nkf1        = 15
  nkf2        = 15
  nkf3        = 15

  nqf1        = 15
  nqf2        = 15
  nqf3        = 15

  use_opt_w2b = .true.

  nbndsub     =  8

  wannierize  = .false.
  num_iter    = 1000
  dis_win_max = 100.0
  dis_froz_max= 8.5
  iprint      = 2
  proj(1)     = 'f=-0.125,-0.125,-0.125:sp3:z=0,0,1:x=0,1,0:zona=2.0'
  proj(2)     = 'f= 0.125, 0.125, 0.125:sp3:z=0,0,1:x=1,0,0:zona=2.0'
  wdata(1)    = 'dis_num_iter = 1000'
  wdata(2)    = 'dis_conv_tol = 1E-10'
  wdata(3)    = 'conv_tol = 1E-10'
  wdata(4)    = 'conv_window = 5'

  nk1         = 2
  nk2         = 2
  nk3         = 2

  nq1         = 2
  nq2         = 2
  nq3         = 2
 /
