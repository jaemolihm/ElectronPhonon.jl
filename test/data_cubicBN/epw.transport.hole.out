                                                                                      
                                       ``:oss/                                        
                           `.+s+.     .+ys--yh+     `./ss+.                           
                          -sh//yy+`   +yy   +yy    -+h+-oyy                           
                          -yh- .oyy/.-sh.   .syo-.:sy-  /yh                           
                 `.-.`    `yh+   -oyyyo.     `/syys:    oys      `.`                  
               `/+ssys+-` `sh+      `                   oys`   .:osyo`                
               -yh- ./syyooyo`                          .sys+/oyo--yh/                
               `yy+    .-:-.                             `-/+/:`  -sh-                
                /yh.                                              oys                 
          ``..---hho---------`   .---------..`      `.-----.`    -hd+---.             
       `./osmNMMMMMMMMMMMMMMMs. +NNMMMMMMMMNNmh+.   yNMMMMMNm-  oNMMMMMNmo++:`        
       +sy--/sdMMMhyyyyyyyNMMh- .oyNMMmyyyyyhNMMm+` -yMMMdyyo:` .oyyNMMNhs+syy`       
       -yy/   /MMM+.`-+/``mMMy-   `mMMh:`````.dMMN:` `MMMy-`-dhhy```mMMy:``+hs        
        -yy+` /MMMo:-mMM+`-oo/.    mMMh:     `dMMN/`  dMMm:`dMMMMy..MMMo-.+yo`        
         .sys`/MMMMNNMMMs-         mMMmyooooymMMNo:   oMMM/sMMMMMM++MMN//oh:          
          `sh+/MMMhyyMMMs- `-`     mMMMMMMMMMNmy+-`   -MMMhMMMsmMMmdMMd/yy+           
    `-/+++oyy-/MMM+.`/hh/.`mNm:`   mMMd+/////:-.`      NMMMMMd/:NMMMMMy:/yyo/:.`      
   +os+//:-..-oMMMo:--:::-/MMMo. .-mMMd+---`           hMMMMN+. oMMMMMo. `-+osyso:`   
   syo     `mNMMMMMNNNNNNNNMMMo.oNNMMMMMNNNN:`         +MMMMs:`  dMMMN/`     ``:syo   
   /yh`     :syyyyyyyyyyyyyyyy+.`+syyyyyyyyo:`         .oyys:`   .oyys:`        +yh   
   -yh-        ````````````````    `````````              ``        ``          oys   
   -+h/------------------------::::::::://////++++++++++++++++++++++///////::::/yd:   
   shdddddddddddddddddddddddddddddhhhhhhhhyyyyyssssssssssssssssyyyyyyyhhhhhhhddddh`   
                                                                                      
  S. Ponce, E. R. Margine, C. Verdi, and F. Giustino,                                 
                                                Comput. Phys. Commun. 209, 116 (2016) 
                                                                                      

     Program EPW v.5.3.1 starts on  2Nov2021 at 12:55:55 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes
     Reading input from epw.transport.hole.in

     WARNING: Mismatch between temps(:) and nstemp

     WARNING: Using supplied temperature list and ignoring nstemp
                      
     ------------------------------------------------------------------------ 
                   RESTART - RESTART - RESTART - RESTART                         
     Restart is done without reading PWSCF save file.                  
     Be aware that some consistency checks are therefore not done.                  
     ------------------------------------------------------------------------ 
                      

     --                                                                         

     bravais-lattice index     =            0
     lattice parameter (a_0)   =       0.0000  a.u.
     unit-cell volume          =       0.0000 (a.u.)^3
     number of atoms/cell      =            0
     number of atomic types    =            0
     kinetic-energy cut-off    =       0.0000  Ry
     charge density cut-off    =       0.0000  Ry
     Exchange-correlation= not set
                           (  -1  -1  -1  -1  -1  -1  -1)


     celldm(1)=    0.00000  celldm(2)=    0.00000  celldm(3)=    0.00000
     celldm(4)=    0.00000  celldm(5)=    0.00000  celldm(6)=    0.00000

     crystal axes: (cart. coord. in units of a_0)
               a(1) = (  0.0000  0.0000  0.0000 )  
               a(2) = (  0.0000  0.0000  0.0000 )  
               a(3) = (  0.0000  0.0000  0.0000 )  

     reciprocal axes: (cart. coord. in units 2 pi/a_0)
               b(1) = (  0.0000  0.0000  0.0000 )  
               b(2) = (  0.0000  0.0000  0.0000 )  
               b(3) = (  0.0000  0.0000  0.0000 )  


     Atoms inside the unit cell: 

   Cartesian axes

     site n.  atom      mass           positions (a_0 units)

 
     No symmetry!

     G cutoff =    0.0000  (      0 G-vectors)     FFT grid: (  0,  0,  0)
     number of k points=    0
                       cart. coord. in units 2pi/a_0
     EPW          :      0.00s CPU      0.00s WALL

     EPW          :      0.00s CPU      0.00s WALL


     -------------------------------------------------------------------
     Using bn.ukk from disk
     -------------------------------------------------------------------

     Symmetries of Bravais lattice:  48
     Symmetries of crystal:          24

     Do not need to read .epb files; read .fmt files


     Band disentanglement is used: nbndsub =    8

     Computes the analytic long-range interaction for polar materials [lpolar]
      
     Construct the Wigner-Seitz cell using Wannier centers and atomic positions 
     Number of WS vectors for electrons      123
     Number of WS vectors for phonons       25
     Number of WS vectors for electron-phonon       25
     Maximum number of cores for efficient parallelization       50

     Reading Hamiltonian, Dynamical matrix and EP vertex in Wann rep from file


     Reading interatomic force constants

     Read Z* and epsilon
     IFC last    0.0091940
      Imposed simple ASR

     Finished reading ifcs


     Finished reading Wann rep data from file

     ===================================================================
     Memory usage:  VmHWM =        62Mb
                   VmPeak =      2611Mb
     ===================================================================
     
     Using uniform q-mesh:   15  15  15
     Size of q point mesh for interpolation:       3375
     Using uniform k-mesh:   15  15  15
     Size of k point mesh for interpolation:       6750
     Max number of k points per pool:             6750

     Fermi energy coarse grid =  10.921574 eV

     ===================================================================

     Fermi energy is read from the input file: Ef =  10.900000 eV

     ===================================================================

              ibndmin =     2  ebndmin =    10.522 eV
              ibndmax =     4  ebndmax =    10.922 eV


     Number of ep-matrix elements per pool :       182250 ~=    1.39 Mb (@ 8 bytes/ DP)

     File ./Fepmatkq1/bn.epmatkq1_0 deleted, as requested
      
     A selecq.fmt file was found but re-created because selecqread == .FALSE. 
     Number selected, total            100            274
     Number selected, total            200            691
     Number selected, total            300           2489
     Number selected, total            400           3089
     We only need to compute      495 q-points
      
     Valence band maximum    =  10.921574 eV
     Conduction band minimum =  15.371436 eV

     Temperature  200.000 K
     Mobility VB Fermi level =  11.127307 eV

     Temperature  300.000 K
     Mobility VB Fermi level =  11.232810 eV

     Temperature  400.000 K
     Mobility VB Fermi level =  11.343188 eV

     ===================================================================
     Scattering rate for IBTE
     ===================================================================

     Restart and restart_step inputs deactivated (restart point at every q-points).
     No intermediate mobility will be shown.

     Fermi Surface thickness =   0.400000 eV
     This is computed with respect to the fine Fermi level  10.900000 eV
     Only states between  10.500000 eV and  11.300000 eV will be included

     Save matrix elements larger than threshold:   0.975461057766E-24
      
     Progression iq (fine) =        100/       495
     Progression iq (fine) =        200/       495
     Progression iq (fine) =        300/       495
     Progression iq (fine) =        400/       495
      200.000     11.1273       0.100000E+16
      300.000     11.2328       0.100001E+16
      400.000     11.3432       0.100000E+16
      
     epmatkqread automatically changed to .TRUE. as all scattering have been computed.
      
     ===================================================================
     Memory usage:  VmHWM =        98Mb
                   VmPeak =      2766Mb
     ===================================================================

     Number of elements per core      43273
      
     =============================================================================================
     BTE in the self-energy relaxation time approximation (SERTA)
     =============================================================================================

     =============================================================================================
       Temp     Fermi   Hole density  Population SR                  Hole mobility 
        [K]      [eV]     [cm^-3]      [h per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  11.1273   0.10000E+16  -0.14484E-08    0.125944E+01    0.865760E-01    0.296743E-01
                                                      0.865760E-01    0.122832E+01    0.143106E+00
                                                      0.296743E-01    0.143106E+00    0.107778E+01
      300.000  11.2328   0.10000E+16  -0.47272E-09    0.611312E+01    0.501977E+00    0.182462E+00
                                                      0.501977E+00    0.593903E+01    0.783224E+00
                                                      0.182462E+00    0.783224E+00    0.509630E+01
      400.000  11.3432   0.10000E+16  -0.14522E-09    0.101831E+02    0.905950E+00    0.337459E+00
                                                      0.905950E+00    0.987422E+01    0.137435E+01
                                                      0.337459E+00    0.137435E+01    0.837843E+01
      
     =============================================================================================
     Start solving iterative Boltzmann Transport Equation
     =============================================================================================

     Iteration number:         1

     =============================================================================================
       Temp     Fermi   Hole density  Population SR                  Hole mobility 
        [K]      [eV]     [cm^-3]      [h per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  11.1273   0.10000E+16  -0.19479E-08    0.132323E+01    0.824362E-01    0.255344E-01
                                                      0.824362E-01    0.129212E+01    0.147246E+00
                                                      0.255344E-01    0.147246E+00    0.114158E+01
      300.000  11.2328   0.10000E+16  -0.58871E-09    0.649693E+01    0.481153E+00    0.161639E+00
                                                      0.481153E+00    0.632284E+01    0.804048E+00
                                                      0.161639E+00    0.804048E+00    0.548011E+01
      400.000  11.3432   0.10000E+16  -0.17876E-09    0.109247E+02    0.868220E+00    0.299729E+00
                                                      0.868220E+00    0.106159E+02    0.141208E+01
                                                      0.299729E+00    0.141208E+01    0.912008E+01

                                                      0.109247E+02    Max error
     Iteration number:         2

     =============================================================================================
       Temp     Fermi   Hole density  Population SR                  Hole mobility 
        [K]      [eV]     [cm^-3]      [h per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  11.1273   0.10000E+16  -0.31403E-08    0.132814E+01    0.819162E-01    0.250144E-01
                                                      0.819162E-01    0.129702E+01    0.147766E+00
                                                      0.250144E-01    0.147766E+00    0.114648E+01
      300.000  11.2328   0.10000E+16  -0.93261E-09    0.653440E+01    0.477585E+00    0.158070E+00
                                                      0.477585E+00    0.636031E+01    0.807616E+00
                                                      0.158070E+00    0.807616E+00    0.551758E+01
      400.000  11.3432   0.10000E+16  -0.27257E-09    0.110156E+02    0.859952E+00    0.291462E+00
                                                      0.859952E+00    0.107067E+02    0.142035E+01
                                                      0.291462E+00    0.142035E+01    0.921092E+01

                                                      0.908371E-01    Max error
     Iteration number:         3

     =============================================================================================
       Temp     Fermi   Hole density  Population SR                  Hole mobility 
        [K]      [eV]     [cm^-3]      [h per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  11.1273   0.10000E+16  -0.42931E-08    0.132848E+01    0.818612E-01    0.249595E-01
                                                      0.818612E-01    0.129737E+01    0.147821E+00
                                                      0.249595E-01    0.147821E+00    0.114683E+01
      300.000  11.2328   0.10000E+16  -0.12188E-08    0.653804E+01    0.477102E+00    0.157588E+00
                                                      0.477102E+00    0.636395E+01    0.808099E+00
                                                      0.157588E+00    0.808099E+00    0.552122E+01
      400.000  11.3432   0.10000E+16  -0.34519E-09    0.110268E+02    0.858553E+00    0.290062E+00
                                                      0.858553E+00    0.107179E+02    0.142175E+01
                                                      0.290062E+00    0.142175E+01    0.922211E+01

                                                      0.111934E-01    Max error
     Iteration number:         4

     =============================================================================================
       Temp     Fermi   Hole density  Population SR                  Hole mobility 
        [K]      [eV]     [cm^-3]      [h per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  11.1273   0.10000E+16  -0.58160E-08    0.132851E+01    0.818559E-01    0.249542E-01
                                                      0.818559E-01    0.129739E+01    0.147826E+00
                                                      0.249542E-01    0.147826E+00    0.114686E+01
      300.000  11.2328   0.10000E+16  -0.15997E-08    0.653841E+01    0.477039E+00    0.157524E+00
                                                      0.477039E+00    0.636431E+01    0.808162E+00
                                                      0.157524E+00    0.808162E+00    0.552159E+01
      400.000  11.3432   0.10000E+16  -0.43725E-09    0.110282E+02    0.858319E+00    0.289828E+00
                                                      0.858319E+00    0.107193E+02    0.142198E+01
                                                      0.289828E+00    0.142198E+01    0.922353E+01

                                                      0.141765E-02    Max error
     Iteration number:         5

     =============================================================================================
       Temp     Fermi   Hole density  Population SR                  Hole mobility 
        [K]      [eV]     [cm^-3]      [h per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  11.1273   0.10000E+16  -0.75846E-08    0.132851E+01    0.818554E-01    0.249537E-01
                                                      0.818554E-01    0.129739E+01    0.147827E+00
                                                      0.249537E-01    0.147827E+00    0.114686E+01
      300.000  11.2328   0.10000E+16  -0.20080E-08    0.653844E+01    0.477031E+00    0.157517E+00
                                                      0.477031E+00    0.636435E+01    0.808170E+00
                                                      0.157517E+00    0.808170E+00    0.552162E+01
      400.000  11.3432   0.10000E+16  -0.52987E-09    0.110284E+02    0.858283E+00    0.289792E+00
                                                      0.858283E+00    0.107195E+02    0.142202E+01
                                                      0.289792E+00    0.142202E+01    0.922371E+01

                                                      0.180978E-03    Max error
     Iteration number:         6

     =============================================================================================
       Temp     Fermi   Hole density  Population SR                  Hole mobility 
        [K]      [eV]     [cm^-3]      [h per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  11.1273   0.10000E+16  -0.97448E-08    0.132851E+01    0.818554E-01    0.249536E-01
                                                      0.818554E-01    0.129739E+01    0.147827E+00
                                                      0.249536E-01    0.147827E+00    0.114686E+01
      300.000  11.2328   0.10000E+16  -0.24886E-08    0.653845E+01    0.477030E+00    0.157516E+00
                                                      0.477030E+00    0.636435E+01    0.808171E+00
                                                      0.157516E+00    0.808171E+00    0.552163E+01
      400.000  11.3432   0.10000E+16  -0.63313E-09    0.110284E+02    0.858277E+00    0.289787E+00
                                                      0.858277E+00    0.107195E+02    0.142202E+01
                                                      0.289787E+00    0.142202E+01    0.922373E+01

                                                      0.233249E-04    Max error
     Iteration number:         7

     =============================================================================================
       Temp     Fermi   Hole density  Population SR                  Hole mobility 
        [K]      [eV]     [cm^-3]      [h per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  11.1273   0.10000E+16  -0.12330E-07    0.132851E+01    0.818554E-01    0.249536E-01
                                                      0.818554E-01    0.129739E+01    0.147827E+00
                                                      0.249536E-01    0.147827E+00    0.114686E+01
      300.000  11.2328   0.10000E+16  -0.30313E-08    0.653845E+01    0.477030E+00    0.157516E+00
                                                      0.477030E+00    0.636435E+01    0.808171E+00
                                                      0.157516E+00    0.808171E+00    0.552163E+01
      400.000  11.3432   0.10000E+16  -0.74304E-09    0.110284E+02    0.858277E+00    0.289786E+00
                                                      0.858277E+00    0.107195E+02    0.142202E+01
                                                      0.289786E+00    0.142202E+01    0.922374E+01

                                                      0.301831E-05    Max error
     Iteration number:         8

     =============================================================================================
       Temp     Fermi   Hole density  Population SR                  Hole mobility 
        [K]      [eV]     [cm^-3]      [h per cell]                    [cm^2/Vs]
     =============================================================================================

      200.000  11.1273   0.10000E+16  -0.15450E-07    0.132851E+01    0.818554E-01    0.249536E-01
                                                      0.818554E-01    0.129739E+01    0.147827E+00
                                                      0.249536E-01    0.147827E+00    0.114686E+01
      300.000  11.2328   0.10000E+16  -0.36552E-08    0.653845E+01    0.477030E+00    0.157516E+00
                                                      0.477030E+00    0.636435E+01    0.808171E+00
                                                      0.157516E+00    0.808171E+00    0.552163E+01
      400.000  11.3432   0.10000E+16  -0.86252E-09    0.110284E+02    0.858277E+00    0.289786E+00
                                                      0.858277E+00    0.107195E+02    0.142202E+01
                                                      0.289786E+00    0.142202E+01    0.922374E+01

                                                      0.392535E-06    Max error

     Unfolding on the coarse grid
     elphon_wrap  :      0.01s CPU      0.01s WALL (       1 calls)

     INITIALIZATION: 




     Electron-Phonon interpolation
     ephwann      :     45.65s CPU     47.88s WALL (       1 calls)
     ep-interp    :     44.15s CPU     46.33s WALL (     495 calls)

     wigner_seitz :      0.31s CPU      0.31s WALL (       1 calls)
     rgd_blk_fine :      0.24s CPU      0.25s WALL (    5550 calls)
     DynW2B       :      0.06s CPU      0.07s WALL (     495 calls)
     HamW2B       :     33.03s CPU     34.80s WALL ( 1677375 calls)
     vmeW2B       :      0.80s CPU      0.83s WALL (   14625 calls)
     ephW2Bp      :      1.27s CPU      1.29s WALL (     495 calls)
     ephW2B       :      0.21s CPU      0.21s WALL (    5625 calls)
     print_ibte   :      2.40s CPU      2.68s WALL (     495 calls)


     MPI routines


     Total program execution
     EPW          :     45.66s CPU     47.89s WALL

     ===============================================================================
     The functionality-dependent EPW.bib file was created with suggested citations. 
     Please consider citing the papers listed in EPW.bib.                           
     ===============================================================================
                                                                                    
